static/js/pm_list.js

```diff
@@ -117,7 +117,16 @@ exports.update_private_messages = function () {
     if (private_messages_open) {
         const container = ui.get_content_element($("#private-container"));
         const new_dom = exports._build_private_messages_list();
-        vdom.update(container, new_dom, prior_dom);
+
+        function insert(html) {
```

Comment:
Why the name `insert`?  I would think something like `replace_content` would be clearer.

Reply:
fixed